data-catalog/base/hue/kustomization.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - hive-site-xml-secret.yaml
+  - hue-db-init-job.yaml
+  - hue-dc.yaml
+  - hue-ini-secret.yaml
+  - hue-mysql-dc.yaml
+  - hue-mysql-pvc.yaml
+  - hue-mysql-secret.yaml
+  - hue-mysql-service.yaml
+  - hue-mysqld-exporter-secret.yaml
+  - hue-route.yaml
+  - hue-secret.yaml
+  - hue-service.yaml
+
+commonLabels:
+  app: hue
+
+configurations:
+  - image-transformer-configuration.yaml
+
+images:
+  - name: hue
+    newName: shgriffi/hue
+    newTag: test
+  - name: mysql
+    newName: openshift/mysql
```

Comment:
Hey @tumido 
I am getting an error with this image
```
The DeploymentConfig "hue-mysql" is invalid: spec.triggers[0].imageChangeParams.from.name: Invalid value: "openshift/mysql:5.7": may not contain '/'
```

Reply:
@maulikjs Hm.. I was afraid this won't work. :confused:  The `DeploymentConfig` kind is really stubborn in that. I wonder though, how come did it work for you in the https://github.com/AICoE/idh-manifests/pull/29 ? It should be doing the same transformation for Postgres DC `imageChange` trigger, isn't it?

This PR:
```yaml
$ kustomize build --enable_alpha_plugins data-catalog/overlays/dev
...
apiVersion: v1
kind: DeploymentConfig
metadata:
  name: hue-mysql
spec:
...
  triggers:
  - imageChangeParams:
      automatic: true
      containerNames:
      - mysql
      from:
        kind: ImageStreamTag
        name: openshift/mysql:5.7
    type: ImageChange
```

Compared to Thriftserver PR:

```yaml
$ kustomize build --enable_alpha_plugins data-catalog/overlays/dev
...
---
apiVersion: v1
kind: DeploymentConfig
metadata:
  name: thriftserver
spec:
...
  triggers:
  - imageChangeParams:
      automatic: true
      containerNames:
      - postgresql
      from:
        kind: ImageStreamTag
        name: openshift/postgresql:9.6
    type: ImageChange
...
```

The `<namespace>/<name>:<tag>` format in the `from` is the cause of the error you're getting. DeploymentConfig strictly requires `<name>:<tag>` only as a name, and namespace as a separate field. Which is different to all other Kubrnetes resources where it can be listed all together. :-1: 

We can workaround it for now by a JSON patch, so the result would be:

```yaml
  - imageChangeParams:
      automatic: true
      containerNames:
      - mysql
      from:
        kind: ImageStreamTag
        name: mysql:5.7
        namespace: openshift
    type: ImageChange
```

Do we need this change in the ThriftServer PR as well or is it working in there? It would be weird if it worked in one place and doesn't in another.

Reply:
Ok this is weird,
It deployed the first time I tried it, BUT I just ran it again on the same cluster and got this error for the other MR: 
```
The DeploymentConfig "postgres" is invalid: spec.triggers[0].imageChangeParams.from.name: Invalid value: "openshift/postgresql:9.6": may not contain '/'
```

Reply:
It might be something to do with my dev cluster but either way lets just change it in both the places

Reply:
Good, fixed via https://github.com/AICoE/idh-manifests/pull/27/commits/08556f3cad347998edb5708becbaec4738f2ceac